omniscidb/L0Mgr/L0Mgr.cpp

```diff
@@ -114,6 +114,10 @@ ze_command_list_handle_t L0CommandList::handle() const {
 
 L0CommandList::~L0CommandList() {
   // TODO: maybe return to pool
+  auto status = zeCommandListDestroy(handle_);
+  if (status) {
+    std::cerr << "Non-zero status for command list destructor" << std::endl;
```

Comment:
Why not LOG(ERROR)?

Reply:
No particular reason, just to stay consistent.

Reply:
I see. I still think moving from `std::err` to `LOG(ERROR)` in this file would be the right thing to do. In tests, ERROR log should be printed to stderr anyway.

Reply:
Done